clients/benchmarks/client.cpp

```diff
@@ -813,7 +813,7 @@ try
         throw std::invalid_argument("Invalid value for --initialization " + initialization);
 
     arg.activation_type = string_to_hipblaslt_activation_type(activation_type);
-    if(arg.activation_type == static_cast<hipblaslt_activation_type>(0))
+    if(arg.activation_type == static_cast<hipblaslt_activation_type>(-1))
```

Comment:
string_to_hipblaslt_activation_type would never return -1

Reply:
Thanks for catching that, fixed it in the latest commit.